src/movies.js

```diff
@@ -1,25 +1,90 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+function getAllDirectors(movies) {
+  const directors = movies.map(function (movie){
+    return movie.directors;
+  })
+}
```

Comment:
Your function isn't returning anything: in the tests, you get 'function should return an array'. Were you having trouble finding out why is it giving you that error?